### PR TITLE
DOC: Fix docstring for tsplot

### PR DESCRIPTION
**Pull request by TomDonoghue**
Browsing through pandas, I noticed a small plotting function who's docs were incomplete / out of date. This is a small update to the docstring for tsplot in plotting/_timeseries. 

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/plotting/_timeseries.py [hunk 1]
```diff
@@ -22,16 +22,22 @@
...
-
-    Notes
-    _____
-    Supports same kwargs as Axes.plot
+    plotf : function
+        Function to plot the given Series.
+    ax : matplotlib axes object, optional, default=None
+        Axis to plot upon. If none, plots on current axis.
+    kwargs
+        Additional keywords arguments passed to plotf.
```
Comment (jschendel) [inline pandas/plotting/_timeseries.py]: "keywords arguments" -> "keyword arguments"

File: pandas/plotting/_timeseries.py [hunk 2]
```diff
@@ -22,16 +22,22 @@
 
 def tsplot(series, plotf, ax=None, **kwargs):
     """
-    Plots a Series on the given Matplotlib axes or the current axes
+    Plots a Series on the given Matplotlib axes or the current axis.
```
Comment (TomAugspurger) [inline pandas/plotting/_timeseries.py]: I *think* that matplotlib uses the phrase "current axes", so the old way was correct here. "axis" is for the "x" or "y" axis of an axes :)

Good you change that back here and on line 33?

**Comment by TomDonoghue** [comment]
@TomAugspurger :
Yep, I think you're right, sorry about that. Fixed!

**Comment by jorisvandenbossche** [comment]
Not that I don't want better docstrings (@TomDonogue thanks for contributing on that part!), but shouldn't this function rather be removed/deprecated? 

We don't use it internally, and it is not exposed in the `pandas.plotting` namespace (it *is* still exposed in `pandas.tseries.plotting`). 
And I also don't think it is actually a useful function for users? (they would just do `ts.plot()`, and here you have to awkwardly provide an actual plotting function)

**Comment by TomDonoghue** [comment]
@jorisvandenbossche 
I was basically poking through pandas, looking for an easy initial PR, so other than noticing that this function is exposed through ts.plotting, I don't have much experience with it. I'm not super clear on the use case, and whether it's all that useful. I tested it out, but it does seem like an awkward function, given you have to pass in a plotting function. 

Point being, if it's better to deprecate, that makes sense to me - let me know if there is anything I can / should do on my end.

**Comment by jreback** [comment]
@TomDonoghue can you deprecate this function? add a test for the deprecation & mention in the whatsnew for 0.22

**Comment by jreback** [comment]
closing as stale. if you want to re-visit this pls ping.

**Comment by jorisvandenbossche** [comment]
@TomDonoghue Still welcome to do a PR to deprecate the function instead! (opened https://github.com/pandas-dev/pandas/issues/18627 for this)

**Pull request closed without merging**